src/components/AppArkApiTable/AppArkApiTable.tsx

```diff
@@ -54,23 +69,52 @@ export function PaginatedSortableTable<T>(props: PaginatedSortableTableProps<T>)
     externalFilters,
     externalFiltersState,
     isSortable = true,
+    setRowSelection,
+    selectedRows,
+    pageSize: pageSizeProp,
+    getRowId,
+    defaultSorting,
+    skip = false,
+    extractPagination,
+    tableKey,
   } = props;
   const [{ pageIndex, pageSize }, setPagination] = useState<PaginationState>({
     pageIndex: 1,
-    pageSize: 10,
+    pageSize: pageSizeProp ?? 10,
   });
-
-  const [sortingState, setSorting] = useState<SortingState>([{ id: "", desc: false }]);
-
+  const reduxTableState = useAppSelector(state => getTableState(state, tableKey ?? ""));
+  const filtersAreEquals = JSON.stringify(reduxTableState?.filters) === JSON.stringify(externalFiltersState);
+  const dispatch = useAppDispatch();
+  const [sortingState, setSorting] = useState<SortingState>(defaultSorting ?? [{ id: "", desc: false }]);
```

Comment:
this looks to me that we're reimplementing Tanstack. Tanstack already have a Sorting and Paging tracking why we need it twice?